src/amp.js

```diff
@@ -63,19 +63,17 @@ try {
 
       maybeValidate(window);
       makeBodyVisible(document, waitForExtensions(window));
-    } catch (e) {
-      makeBodyVisible(document);
     } finally {
+      makeBodyVisible(document);
       perf.tick('e_is');
       // TODO(erwinm): move invocation of the `flush` method when we have the
       // new ticks in place to batch the ticks properly.
       perf.flush();
     }
   }, /* opt_isRuntimeCss */ true);
-} catch (e) {
```

Comment:
Revert this change. We overthought it. It should not always make it visible syncronously.